python/feathub/processors/flink/table_builder/aggregation_utils.py

```diff
@@ -41,12 +41,16 @@ def __init__(
         expr: str,
         agg_func: AggFunc,
         window_size: timedelta,
+        limit: int,
+        filter_expr: str,
```

Comment:
Given that this file is used in both sliding windows and over windows, it might be better to add some description or check that over window is not using these fields.

Reply:
I added a TODO to optimize the over window to support different window sizes, limit, and filter expr.